src/components/Button/Button.scss

```diff
@@ -247,27 +247,58 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
 // stylelint-disable selector-max-specificity
 
 .plain {
-  @include recolor-icon(color('blue'));
+  @include recolor-icon(var(--p-interactive-action, color('blue')));
   margin: (-1 * $vertical-padding) rem(-8px);
   padding-left: spacing(tight);
   padding-right: spacing(tight);
   background: transparent;
   border: 0;
   box-shadow: none;
-  color: color('blue');
+  color: var(--p-interactive-action, color('blue'));
+
+  .Content {
+    font-weight: 400;
```

Comment:
This applies to both current and new theming?

Reply:
It does. This does not represent a change from the existing implementation (since it defaulted to this weight), and is needed to override font weight applied to all buttons.